SConstruct

```diff
@@ -3,44 +3,63 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import os
+import sys
 import SCons
 
 opts = Variables()
 opts.AddVariables(
     BoolVariable("DEBUG", "Make debug build", 0),
     BoolVariable("SANITIZE", "Use AddressSanitizer and UBSanitizer", 0),
-    BoolVariable("VERBOSE", "Show full build information", 0))
+    BoolVariable("VERBOSE", "Show full build information", 0),
+)
 
-env = Environment(options = opts,
-                  ENV = os.environ)
+env = Environment(options=opts, ENV=os.environ)
 
-env.Tool('clang')
+env.Tool("clang")
 
 # Add extra compiler flags from the environment
 for flag in ["CCFLAGS", "CFLAGS", "CPPFLAGS", "CXXFLAGS", "LINKFLAGS"]:
-  if flag in os.environ:
-    env.Append(**{flag: SCons.Util.CLVar(os.getenv(flag))})
+    if flag in os.environ:
+        env.Append(**{flag: SCons.Util.CLVar(os.getenv(flag))})
 
-if env["DEBUG"]: 
+if env["DEBUG"]:
     print("DEBUG MODE!")
-    env.Append(CCFLAGS = ["-g", "-DDEBUG"])
+    env.Append(CCFLAGS=["-g", "-DDEBUG"])
 
 if env["SANITIZE"]:
-    sanitizers = ['-fsanitize=address,undefined', \
-                  '-fno-sanitize-recover=undefined,integer,nullability']
-    env.Append(CCFLAGS = sanitizers, LINKFLAGS = sanitizers)
+    sanitizers = [
+        "-fsanitize=address,undefined",
+        "-fno-sanitize-recover=undefined,integer,nullability",
+    ]
+    env.Append(CCFLAGS=sanitizers, LINKFLAGS=sanitizers)
 
-env.Append(
-  LIBS = ["mprio", "mpi", "nss3", "nspr4"],
-  LIBPATH = ['#build/prio', "#build/mpi"],
-  CFLAGS = [ "-Wall", "-Werror", "-Wextra", "-O3", "-std=c99", "-Wvla",
-    "-I/usr/include/nspr", "-I/usr/include/nss", "-Impi", "-DDO_PR_CLEANUP"])
+if sys.platform == "darwin":
+    env.Append(LINKFLAGS=["-L/usr/local/opt/nss/lib"])
```

Comment:
Hm, I guess we're baking in the idea that people are using homebrew, but that's fine I suppose.